### PR TITLE
[EasyStandard] Added new rectors and sniffs

### PR DESCRIPTION
**Pull request by BOB41K1987**
- Add new rules for quality tools

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes  
| Fixed tickets |

### CODE REVIEW

File: packages/EasyStandard/tests/Sniffs/ControlStructures/ArrangeActAssertSniffTest.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+use EonX\EasyStandard\Sniffs\ControlStructures\ArrangeActAssertSniff;
+use Symplify\EasyCodingStandardTester\Testing\AbstractCheckerTestCase;
+
+/**
+ * @covers \EonX\EasyStandard\Sniffs\ControlStructures\ArrangeActAssertSniff
+ *
+ * @internal
+ */
+final class ArrangeActAssertSniffTest extends AbstractCheckerTestCase
```
[the remark this answers is not in the document]
Comment (BOB41K1987) [reply packages/EasyStandard/tests/Sniffs/ControlStructures/ArrangeActAssertSniffTest.php:15]: We can inline `$privatePropertyValue` - this is the easiest solution by far.

File: packages/EasyStandard/src/Sniffs/Exceptions/ThrowExceptionMessageSniff.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        if ($messageToken['code'] === \T_VARIABLE) {
+            return;
+        }
+
+        if ($messageToken['code'] === \T_CLOSE_PARENTHESIS) {
+            return;
+        }
+
+        if (Strings::startsWith($messageToken['content'], "'exceptions.") === false) {
```
Comment (natepage) [inline packages/EasyStandard/src/Sniffs/Exceptions/ThrowExceptionMessageSniff.php:44]: Hmm.. Ok I think that this sniff is really opinionated but I can see some good in it 😄 

My only real issue is that all translations are not necessarily gonna start with `exceptions.` that depends on the app. I think that this sniff should accept an optional array of prefixes to test against, if the `$messageToken['content']` doesn't start with at least one of them then the sniff should fail.

You can default this list of prefixes to `['exceptions.']` if you want I don't mind.

Comment (BOB41K1987) [reply packages/EasyStandard/src/Sniffs/Exceptions/ThrowExceptionMessageSniff.php]: Done.

**Comment by TomasVotruba** [comment]
Hi,

I got a tip for this rule form @natepage 

I really like it :+1: 

@roman-eonx  Would you like to add to @rectorphp core? :)

**Comment by roman-eonx** [comment]
Hi @TomasVotruba,

Which rules are you asking about? Because we have multiple ones here.

**Comment by TomasVotruba** [comment]
Oh, I missed that, sorry.

This `StrictInArrayRector`

**Comment by TomasVotruba** [comment]
I checked the rest and also `AddSeeAnnotationRector` seem useful for common developers. I assume its useful in long test to make it easier to jump between test method and the data it tests.

What do you think?

**Comment by roman-eonx** [comment]
@TomasVotruba yes, we could add those ones, as they can be helpful for other developers and are not specific to our projects.

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**